src/game/WorldHandlers/Map.cpp

```diff
@@ -658,6 +658,9 @@ void Map::Remove(Player* player, bool remove)
     SendRemoveTransports(player);
     UpdateObjectVisibility(player, cell, p);
 
+#ifdef ENABLE_PLAYERBOTS
+    if (!player->GetPlayerbotAI())
+#endif
     player->ResetMap();
```

Comment:
Could we wrap this to be if, else so that the IF isnt dangling like that?
```
#ifdef ENABLE_PLAYERBOTS
    if (!player->GetPlayerbotAI())
        player->ResetMap();
#else
    player->ResetMap();
#endif
```